skycoin/skywire
# Feature/runtime logs

Did you run `make format && make check`?

Fixes #725 

 Changes:	
- Add `GET /api/visors/{pk}/runtime-logs` endpoint

How to test this PR:
Run visor with hypervisor. Check output of the request.

Response is an array of json-encoded log entries. Entries are created by logger clients and can have custom fields with values, e.g. PK field that provide additional info.
Each log entry should have the following fields:
`time`: formatted timestamp of the log entry
`msg`: message of the entry
`level`: level of the entry
`log_line`: log entry line in the global log. Since the capacity of log store is limited, some old entries could be dropped.
All other keys may be absent in some entries.
